src/local-ssh/common.ts

```diff
@@ -0,0 +1,97 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Gitpod. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import winston from 'winston';
+import { existsSync, unlinkSync } from 'fs';
+import { tmpdir } from 'os';
+import { join } from 'path';
+import { GetWorkspaceAuthInfoResponse } from '../proto/typescript/ipc/v1/ipc';
+
+export const PID_LOCK_FILE = '/tmp/gp-daemon.lock';
```

Comment:
this is unsafe

Reply:
I'm going remove this `.lock` file as it's useless, we can check it via local ssh server's port used or not